### PR TITLE
CONSOLE-2442: i18n for replicasets

### PR DESCRIPTION
**Pull request by zherman0**
Add i18n to ReplicaSets page

[CONSOLE-2442](https://issues.redhat.com/browse/CONSOLE-2442)

### CODE REVIEW

**Comment by zherman0** [comment]
/hold
I have some questions before I want this to merge.

File: frontend/public/components/replicaset.jsx
Comment (spadgett) [inline frontend/public/components/replicaset.jsx:49]: We want to use sentence case for almost all headings and labels

```suggestion
        <SectionHeading text={t('workload~ReplicaSet details')} />
```

File: frontend/public/components/replicaset.jsx
Comment (spadgett) [inline frontend/public/components/replicaset.jsx:55]:
```suggestion
                  <dt>{t('workload~Deployment revision')}</dt>
```

Comment (spadgett) [inline frontend/public/components/replicaset.jsx:55]:
```suggestion
                  <dt>{t('workload~Deployment revision')}</dt>
```

File: frontend/package.json
Comment (spadgett) [inline frontend/package.json:266]: Was this unintentional? (Good to update this, but we should do it as a separate PR.)

Comment (zherman0) [reply frontend/package.json:266]: This was intentional; It is needed for latest version of python on my system.  I will pull it out and place in a new PR.

**Review by spadgett — APPROVED** [review]
/lgtm
/hold

/assign @yapei @ahardin-rh @sferich888 

This is ready for QE review.

**Comment by spadgett** [comment]
@zherman0 FYI, needs rebase

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**
